transport/grpc/stream_test.go

```diff
@@ -368,6 +374,43 @@ func TestStreaming(t *testing.T) {
 				),
 			),
 		},
+
+		{
+			name: "stream with error details",
+			services: Lifecycles(
+				GRPCService(
+					Name("myservice"),
+					p.NamedPort("13"),
+					Proc(
+						Name("proc"),
+						OrderedStreamHandler(
+							RecvStreamMsg("test"),
+							RecvStreamErr(io.EOF.Error()),
+							SendStreamMsg("test1"),
+							SendStreamMsg("test2"),
+							SendStreamMsg("test3"),
```

Comment:
for the simplification of this unittest, let's remove all the unneeded steps:

```
RecvStreamMsg("test"),
RecvStreamErr(io.EOF.Error()),
SendStreamMsg("test1"),
SendStreamMsg("test2"),
SendStreamMsg("test3"),
```
Right now, this test could fail if something else than details is broken, let's test only one unit of the stream feature here

Reply:
Have kept a single ping/pong followed by an error, removed rest of the steps